src/components/SearchMessages.vue

```diff
@@ -184,20 +184,20 @@
 						{{ t('mail', 'Tags') }}
 					</label>
 					<div class="modal-inner--container">
-						<NcMultiselect
+						<NcSelect
 							v-if="tags.length > 0"
 							id="tagsId"
 							v-model="selectedTags"
 							class="multiselect-search-tags"
 							:options="tags"
-							label="displayName"
+							label="name"
```

Comment:
why does it change?

Reply:
thats a copy paste issue. it should be as it was